src/bundle/Resources/views/themes/admin/ui/component/alert/alert.html.twig

```diff
@@ -1,14 +1,22 @@
 {% import "@ibexadesign/ui/component/macros.html.twig" as html %}
 
 {% set icon_name_map = {
-    info: 'system-information',
-    error: 'circle-close',
-    warning: 'warning-triangle',
-    success: 'checkbox',
+    info: 'about',
+    error: 'notice',
+    warning: 'warning',
+    success: 'approved',
 } %}
 
+{% set size_name_map = {
```

Comment:
is it needed, as it's perfect mapping key=>value?

Reply:
It works the same in js